src/pages/Guya/main.ts

```diff
@@ -46,8 +46,9 @@ export const Guya: pageInterface = {
     readerConfig: [
       {
         current: {
-          selector: '.ReaderImageWrapper',
-          mode: 'countAbove',
+          selector: '.rdr-page-selector-counter',
+          mode: 'text',
+          regex: '^\\d+',
         },
         total: {
```

Comment:
Would it not be better in this case to use the .rdr-page-selector-counter for the total too. So that it uses the same place for both?